components/search_results_item/index.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+
+import {getCurrentTeam} from 'mattermost-redux/selectors/entities/teams';
+
+import {closeRightHandSide} from 'actions/views/rhs';
+
+import SelectResultsItem from './search_results_item.jsx';
+
+function mapStateToProps(state, ownProps) {
+    return {
+        ...ownProps,
```

Comment:
I know we do this everywhere, but it's actually not needed since it's automatically for us. Mind removing this line?

Reply:
Ya of course